### DISCUSSION TITLE
How to upload files to Supabase from a SwiftUI app?

### DISCUSSION THREAD

**Discussion by cwilharm** [supabase/supabase#8367] [Questions] [answered]
Hey there, I want to connect my SwiftUI app to Supabase Storage (I want to upload images from my app to Supabase). Does anyone have an idea how i can do this? 
Thanks in advance!

**Comment by fenos** [comment, marked as answer]
You can simply send an HTTP request to the [upload endpoint](https://supabase.github.io/storage-api/#/object/post_object__bucketName___wildcard_) 

If you have any questions feel free to reach out

**Reply by cwilharm** [reply]
Ok thanks a lot! Can you help me with that HTTP request in SwiftUI? I have absolutely no idea how to code this…
